### ISSUE TITLE
Uncaught SyntaxError in js.php

### ISSUE THREAD

**Issue by kmlucy** [splitbrain/dokuwiki#1779]
I believe https://github.com/LotarProject/dokuwiki-template-bootstrap3/issues/257 is actually a dokuwiki bug, and not a bootstrap3 bug. The gist of it is that js.php seems to be cutting off too early. Here is the last few lines of ```js.php?t=dokuwiki&tseed=095a61482e218f8fc683972907c09a9c```

```
	_filterMenuItems: function(character) {
		var escapedCharacter = character.replace( /[\-\[\]{}()*+?.,\\\^$|#\s]/g, "\\$&" ),
			regex = new RegExp( "^" + escapedCharacter, "i" );

		return this.activeMenu
			.find( this.options.items )

			// Only match on items, not dividers or other content (#10571)
			.filter( ".ui-menu-item" )
			.filter(function() {
				return regex.test( $.trim( $( this
```

**Comment by splitbrain** [comment]
js.php should not cut off anything. The only reason I can imagine that happening would be a problem with the compression mechanism. But your code looks uncompressed.

In the original report you mention that depending on switching on and off plugins, the cut off part changes. 

- Can you compare the size of the delivered JavaScript for multiple scenarios (different plugins enabled)? Is it always the same?
- Can you check if the cutoff always happens at the same code?
- Does the cutoff only happen with the bootstrap3 template enabled?

**Comment by kmlucy** [comment]
- In the js.php link, the t value changes based on whether I have bootstrap3 enabled or dokuwiki enabled, and the tseed value changes. Right now, the tseed value is tseed=876ccda007c533c54ea41900a658a5c4 and the delivered js.php is 12525 lines. I disabled the info plugin, and now tseed=13b86dd43b46158cc944e0896030112b and it is 10909 lines. After reenabling the info plugin, tseed=4b54518af9974b566f14158a7c187b23 and it is 11622 lines. 
- The cutoff always seems to be at different points in the code. Sometimes it seems line the last line is complete, others it seems like it cuts off midway through a line. 
- The issue occurs no matter what is enabled. I have tried uninstalling bootstrap3, and the problem persists, although not as obvious as js is needed less. I have also tried disabling all the plugins one at a time, and none of them seem to be causing it.

**Comment by splitbrain** [comment]
The tseed changes everytime your config changes. That's fine. Please check the size in bytes not lines.

**Comment by kmlucy** [comment]
Same scenario (info enabled, disabled, enabled again): 312 KB, 263 KB, 316 KB.

**Comment by kmlucy** [comment]
Here are the ends of the file for each case:

1. 
```
	_setOption: function( key, value ) {
		if ( key === "active" ) {
			// _activate() will handle invalid values and update this.options
			this._activate( value );
			return;
		}

		i
```
2. 
```
	var pluses = /\+/g;

	function raw(s) {
		return s;
	}

	function decoded(s) {
		return decodeURIComponent(s.replace(pluses, ' '));
	}

	function converted(s
```
3.
 ```
	_refresh: function() {
		var maxHeight,
			options = this.options,
			heightStyle = options.heightStyle,
			parent = this.element.parent();

		this.active = this._findActive( options.active )
			.addClass( "ui-accordion-header-active ui-state-active ui-corner-top" )
			
```

**Comment by Klap-in** [comment]
Which server/php/engine/version/etc. do you use?

**Comment by kmlucy** [comment]
This is through the https://github.com/istepanov/docker-dokuwiki container. It uses php7 and nginx running on Alpine. 
```
# nginx -v
nginx version: nginx/1.10.1
# php7 -v
PHP 7.0.13 (cli) (built: Nov 16 2016 17:49:30) ( NTS )
Copyright (c) 1997-2016 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2016 Zend Technologies
# cat /etc/*-release
3.4.4
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.4.4
PRETTY_NAME="Alpine Linux v3.4"
HOME_URL="http://alpinelinux.org"
BUG_REPORT_URL="http://bugs.alpinelinux.org"
```

**Comment by kmlucy** [comment]
I solved it. It turns out it was a permissions error on the ```/var/lib/nginx/``` directory, which was resulting in permission being denied to the fastcgi directory. I changed the directory owner and it looks like it is fixed now.

 I believe the issue stemmed somehow from a combination of my work in setting it up, and the Docker image I use: [istepanov/docker-dokuwiki](https://github.com/istepanov/docker-dokuwiki). I have not had the problem resurface even after rebuilding the container, so I'm not sure how it happened. Regardless, it was not an issue with either DokuWiki or Bootstrap3, as I had initially suspected. 

Thank you all for the help troubleshooting,
Kyle
